var/spack/repos/builtin/packages/arm-forge/package.py

```diff
@@ -19,107 +19,107 @@ class ArmForge(Package):
     #       versions (and checksums) based on the target platform shows up
 
     version(
-        "19.0.4-linux-rhel6-x86_64",
+        "19.0.4-Redhat-6.0-x86_64",
```

Comment:
This is kind of an unfortunate change in convention.

Reply:
I thought about it, and decided that when we add a `when=` parameter to the `version()` directive, all of these will become 19.0.4 anway, so it didn't really matter what we called the version. I can revert this package if you want, I don't really care.